lib/cronitor/monitor.rb

```diff
@@ -5,6 +5,7 @@ class Monitor
     attr_reader :key, :api_key, :api_version, :env
 
     PING_RETRY_THRESHOLD = 3
+    API_URL = 'https://cronitor.io/api/monitors'.freeze
```

Comment:
Is there a need to make this configurable? Something for internal testing/usage for example.